dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JsonConfigBootstrapTest.java

```diff
@@ -496,10 +503,11 @@ private void runApplication(RemoteFile consoleLogFile) throws Exception {
         TestUtils.runApp(server, "logServlet");
     }
 
-    private static String setServerConfig(String fileName) throws Exception {
-        server.setMarkToEndOfLog();
+    private static void setServerConfig(String fileName) throws Exception {
+        RemoteFile log = server.getDefaultTraceFile();
+        server.setMarkToEndOfLog(log);
         server.setServerConfigurationFile(fileName);
-        return server.waitForStringInLogUsingMark("CWWKG0017I.*|CWWKG0018I.*");
+        Assert.assertNotNull(server.waitForStringInLog("CWWKG0017I.*|CWWKG0018I.*", log));
```

Comment:
Is trace enabled when this method is called?

Reply:
Trace should be enabled by default to `*=info:logservice=detail:com.ibm.ws.runtime.update.*=all`. None of the servers have an empty `traceSpecification` like `traceSpecification=""`, so there should be trace enabled for every server config update.

https://github.com/OpenLiberty/open-liberty/blob/76bc2fe98fba68f608914facb96ee8b99c69531c/dev/fattest.simplicity/autoFVT-defaults/testports.properties#L145-L146